### DISCUSSION TITLE
anyone had any luck with getting a VM.Standard.A1.Flex instance using this script?

### DISCUSSION THREAD

**Discussion by infinitephantasm** [mohankumarpaluru/oracle-freetier-instance-creation#53] [General]
been trying to get a proper `VM.Standard.A1.Flex` instance on ashburn for a really long time; more than a month at the least, with some intermittent downtime, to no success.

initially i'd just leave an ssh window open with it running, but eventually i just opted to `tmux` it and have it run that way in case i needed to restart my PC (or if it crashed for some reason).

it's been about a month since i started using the script (first ran on october 4th 2024), with it making attempts every minute or so, and it doesn't seem like i've got any closer to an instance, with all the info in the `launch_instance.log` just mentioning `Out of host capacity.`, with the very very rare `An internal error has occurred.`

am i just really, *really* unlucky, or is this just a hopeless endeavor for the region i've chosen?

![image](https://github.com/user-attachments/assets/3ca59af7-0d62-4636-80cc-3ef7f1e36f13)

**Comment by infinitephantasm** [comment]
![IMG_1825](https://github.com/user-attachments/assets/3ec11a83-a9cd-4353-8f81-aaa2bb10aeb9)
![IMG_1824](https://github.com/user-attachments/assets/89edb270-8346-44e2-80cb-e13de9af8351)

update: it took over 2 months for me. might take more, might take less.

just be aware that this will likely be a Very long venture, and that you probably want to set up some kind of idle process on your micro server so it doesn't get nuked

**Comment by megvadulthangya** [comment]
I have same question. Anyone has sucessfully use this script ? in 2025??
